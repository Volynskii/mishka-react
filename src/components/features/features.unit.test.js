import React from "react";
import Features from "./features";
import renderer from "react-test-renderer";
import {shallow} from "enzyme";

describe('Features component', ()=> {
    it('should render Features Component', ()=> {
        const dom = <Features />;
        const component = renderer.create(dom);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it(' Features Component should have 6 items', ()=> {
        const dom = shallow(<Features />);
        const link = dom.find(".features__list__item");
        expect(link).toHaveLength(6)
    });
});
