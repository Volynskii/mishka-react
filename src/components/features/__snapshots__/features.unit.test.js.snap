// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features component should render Features Component 1`] = `
<article
  className="features"
>
  <span>
    Коротко о нас:
  </span>
  <section>
    <ul
      className="features__list"
    >
      <li
        className="features__list__item"
      >
        <div
          className="features__list__item__picture--one"
        />
        <div>
          Экологически чистые материалы
        </div>
      </li>
      <li
        className="features__list__item"
      >
        <div
          className="features__list__item__picture--two"
        />
        <div>
          Связано вручную с любовью и умилением
        </div>
      </li>
      <li
        className="features__list__item"
      >
        <div
          className="features__list__item__picture--three"
        />
        <div>
          Скандинавский стиль по российской цене
        </div>
      </li>
      <li
        className="features__list__item"
      >
        <div
          className="features__list__item__picture--four"
        />
        <div>
          Поддержка отечественного производителя
        </div>
      </li>
      <li
        className="features__list__item"
      >
        <div
          className="features__list__item__picture--five"
        />
        <div>
          Увеличивает лайки на фотографиях
        </div>
      </li>
      <li
        className="features__list__item"
      >
        <div
          className="features__list__item__picture--six"
        />
        <div>
          Поставляется в подарочной упаковке
        </div>
      </li>
    </ul>
  </section>
</article>
`;
